addon/addon-test-support/@ember/test-helpers/-internal/is-component.ts

```diff
@@ -15,15 +14,21 @@ function isComponent(
   maybeComponent: object,
   owner: object
 ): maybeComponent is ComponentLike {
-  if (macroCondition(dependencySatisfies('ember-source', '>=3.25.0-beta.1'))) {
+  // SAFETY: in more recent versions of @glimmer/manager,
+  //         this throws an error when maybeComponent does not have
+  //         an associated manager.
+  try {
```

Comment:
It appears that `@glimmer/manager` changed its behavior. This change absorbs the change in `@glimmer/manager`